ts/lib/auth/googleauth.ts

```diff
@@ -465,36 +478,36 @@ export default class GoogleAuth {
    * Determines whether a file exists. Allows mocking.
    * @api private
    */
-  public _fileExists(filePath) {
+  public _fileExists(filePath: string) {
     return fs.existsSync(filePath);
   }
 
   /**
    * Joins two parts of a path. Allows mocking.
    * @api private
    */
-  public _pathJoin(item1, item2) {
+  public _pathJoin(item1: string, item2: string) {
     return path.join(item1, item2);
   }
 
   /**
    * Allows mocking of the path to a well-known file.
    * @api private
    */
-  private _mockWellKnownFilePath(filePath) {
+  private _mockWellKnownFilePath(filePath: string) {
     return filePath;
   }
 
   // Executes the given callback if it is not null.
-  private callback(c, ...args) {
+  private callback(c: Function, ...args: any[]) {
```

Comment:
I was able to add an `err?: Error` in the middle there, but this method is used with a varying number of varying types after that.  There are `n` arguments, so it's hard to figure out what the types would be.